controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -666,3 +476,205 @@ stream {
         add_header 'Access-Control-Allow-Headers' 'DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Authorization';
      }
 {{ end }}
+
+{{/* definition of server-template to avoid repetitions with server-alias */}}
+{{ define "SERVER" }}
+        listen 80{{ if $.Cfg.UseProxyProtocol }} proxy_protocol{{ end }}{{ if eq .Hostname "_"}} default_server reuseport backlog={{ $.BacklogSize }}{{end}};
```

Comment:
Due to the scope being passed into the template ($server) then it will not be able to find $.Cfg...

I currently am working on resolving this.